Swerve/SwerveModules/SwerveModuleIO.java

```diff
@@ -0,0 +1,106 @@
+package POPLib.Swerve.SwerveModules;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import org.littletonrobotics.junction.AutoLog;
+import java.lang.Cloneable;
+import java.lang.Override;
+import org.littletonrobotics.junction.LogTable;
+import org.littletonrobotics.junction.inputs.LoggableInputs;
+
+
+public interface SwerveModuleIO {
+    @AutoLog
+    public static class ModuleIOInputsAutoLogged implements LoggableInputs, Cloneable{
+        public boolean driveConnected = false;
+        public double drivePositionRad = 0.0;
+        public double driveVelocityRadPerSec = 0.0;
+        public double driveAppliedVolts = 0.0;
+        public double driveCurrentAmps = 0.0;
+    
+        public boolean turnConnected = false;
+        public boolean turnEncoderConnected = false;
+        public Rotation2d turnAbsolutePosition = new Rotation2d();
+        public Rotation2d turnPosition = new Rotation2d();
+        public double turnVelocityRadPerSec = 0.0;
+        public double turnAppliedVolts = 0.0;
+        public double turnCurrentAmps = 0.0;
+    
+        public double[] odometryTimestamps = new double[] {};
+        public double[] odometryDrivePositionsRad = new double[] {};
+        public Rotation2d[] odometryTurnPositions = new Rotation2d[] {};
+
+        @Override
+        public void toLog(LogTable table) {
```

Comment:
Autolog annotation already accomplishes this, is there a reason this is being overridden